test/testUtils.js

```diff
@@ -0,0 +1,224 @@
+/*
+ * testutils.js - test the utility functions
+ *
+ * Copyright © 2023 JEDLSoft
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { isKababCase, isCamelCase, isSnakeCase } from '../src/utils.js';
+
+export const testUtils = {
+    testIsKababCaseTrue: function(test) {
+        test.expect(1);
+
+        test.ok(isKababCase("This-is-Kabab-case"));
+
+        test.done();
+    },
+
+    testIsKababCaseDegenerate: function(test) {
+        test.expect(1);
+
+        test.ok(isKababCase("this"));
```

Comment:
As discussed, detection shouldn't return true for degenerate cases or it could lead to various false-positives in rules that use it

Reply:
fixed